frontend/public/components/storage/create-pvc.tsx

```diff
@@ -80,14 +81,20 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
     this.setState({ nameValuePairs }, this.onChange);
   };
 
+  handleCephProvisioner = (sc: string) => {
+    return isCephProvisioner(sc)
+      ? ['ReadWriteOnce', 'ReadWriteMany']
+      : ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'];
+  };
+
   handleStorageClass = (storageClass) => {
     //if the provisioner is unknown or no storage class selected, user should be able to set any access mode
     const modes =
       storageClass &&
       storageClass.provisioner &&
       provisionerAccessModeMapping[storageClass.provisioner]
         ? provisionerAccessModeMapping[storageClass.provisioner]
-        : ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'];
+        : this.handleCephProvisioner(storageClass.provisioner);
```

Comment:
`storageClass` can be null, so this will be a runtime error.

Reply:
ack